src/MgvOfferTaking.sol

```diff
@@ -516,8 +516,13 @@ abstract contract MgvOfferTaking is MgvHasOffers {
         }
 
         /* We update the totals in the multiorder based on the adjusted `sor.wants`/`sor.gives`. */
-        /* overflow: sor.{wants,gives} are on 96bits, sor.total{Got,Gave} are on 256 bits. */
+        /* no overflow: sor.wants is on 96 bits */
```

Comment:
how is sor.wants 96 bits? If fillWants is false, and an offer wants a high amount (say close to close to 256 bits), then in `execute` we have `sor.wants = LogPriceLib.outboundFromInbound(sor.offer.logPrice(), fillVolume);` - fillVolume is bounded by MAX_SAFE_VOLUME which is >>96 bits. What am I missing?

Reply:
Just below you say sor.wants can be 248 bits? :)

Reply:
Thanks fixed in 0aba68666b1773bcb14343cf2dd5af88936ebd96! (I meant sor.gives can be 248 bits)

sor.wants is 96 bits because the line
```
sor.wants = LogPriceLib.outboundFromInbound(sor.offer.logPrice(), fillVolume)
```

basically means `taker.wants = fillVolume/offer.price`, and we know that point that `fillVolume < offer.wants`. Since `offer.wants = offer.gives * offer.price`, we have `taker.wants <= offer.gives` and `offer.gives` is on 96 bits.